### PR TITLE
Prepare internal packages

### PR DESCRIPTION
**Pull request by Lykoi18**

### CODE REVIEW

File: packages/devextreme/js/__internal/core/license/license_validation_internal.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  };
+}
+
+export function validateLicense(): void {}
+
+export function setLicenseCheckSkipCondition(): void {}
+
+export default {
+  validateLicense,
+};
```
Comment (IlyaKhD) [inline packages/devextreme/js/__internal/core/license/license_validation_internal.ts]:
```suggestion
export default {
  validateLicense(): void {}
};
```

Comment (IlyaKhD) [reply packages/devextreme/js/__internal/core/license/license_validation_internal.ts]: I guess all other exports are used only in tests

File: packages/devextreme/build/gulp/npm.js [hunk 2]
```diff
@@ -74,9 +79,24 @@ const distGlobs = distGlobsPattern(ctx.RESULT_JS_PATH);
 
 const jsonGlobs = ['js/**/*.json', '!js/viz/vector_map.utils/*.*'];
 
+const updatePackageName = lazyPipe()
```
Comment (IlyaKhD) [inline packages/devextreme/build/gulp/npm.js]:
```suggestion
const overwriteInternalPackageName = lazyPipe()
```

File: packages/devextreme/js/__internal/core/license/license_validation_internal.ts (hunk 1 above)
Comment (timbset) [reply packages/devextreme/js/__internal/core/license/license_validation_internal.ts]: In my opinion, it is better to leave exports the same regardless its usage

**Review by timbset — DISMISSED** [review]

**Review by IlyaKhD — DISMISSED** [review]

**Review by IlyaKhD — DISMISSED** [review]

File: packages/devextreme/js/__internal/core/license/license_validation_internal.ts (hunk 1 above)
Comment (IlyaKhD) [reply packages/devextreme/js/__internal/core/license/license_validation_internal.ts]: OK

I've added testing for equality of modules signatures

**Review by IlyaKhD — DISMISSED** [review]

**Review by timbset — DISMISSED** [review]

**Review by IlyaKhD — DISMISSED** [review]

**Review by IlyaKhD — APPROVED** [review]

**Pull request merged**